jinja/navbar.html

```diff
@@ -2,7 +2,8 @@
     ('/qml', 'qml', 'Quantum machine learning'),
     ('install.html', 'install', 'Install'),
     ('plugins.html', 'plugins', 'Plugins'),
-    ('https://pennylane.readthedocs.io', 'docs', 'Documentation')
+    ('https://pennylane.readthedocs.io', 'docs', 'Documentation'),
+    ('https://qhack.ai', 'qhack', 'QHACK2021')

```

Comment:
What about putting a mini QHACK logo here?

Reply:
Tried it, it looks ugly imo 😞 

![image](https://user-images.githubusercontent.com/2959003/102594532-49212f00-4151-11eb-81ad-318a3f9c4bd3.png)